src/document/2025/DocumentTypes.ts

```diff
@@ -137,7 +134,6 @@ export interface Output {
 
 export interface Trajectory {
   name: string;
-  version: typeof TRAJ_SCHEMA_VERSION;
```

Comment:
The frontend should still ensure the schema is complete using the type system as much as it can. Removing the version field can cause the `...` syntax for using values from one object to make another not carry over all the data the object may hold.